docs/src/components/Editor/Editor.tsx

```diff
@@ -4,11 +4,8 @@ import * as React from 'react'
 import AceEditor, { AceEditorProps } from 'react-ace'
 import * as ace from 'brace'
 import 'brace/ext/language_tools'
-import 'brace/mode/html'
-import 'brace/mode/json'
 import 'brace/mode/jsx'
-import 'brace/mode/sh'
-import 'brace/theme/tomorrow_night'
+import 'brace/theme/tomorrow_night_eighties'
```

Comment:
I have to change the theme because previous theme from `brace` and `prismjs` theme looked weird before